test/DriverFactoryTest.php

```diff
@@ -42,6 +42,6 @@ public function testItSupportsGlobalBasenameOptionOnFileDrivers()
         $factory = new DriverFactory();
 
         $driver = $factory($container->reveal());
-        $this->assertSame($globalBasename, $driver->getGlobalBasename());
+        static::assertSame($globalBasename, $driver->getGlobalBasename());
```

Comment:
you know what, I'll leave it at that for now, and then make a different PR where we update PHPUnit and standardise assertion invocation all over the test suite.